src/microbe_stage/MicrobeStage.cs

```diff
@@ -514,6 +514,20 @@ public override void OnReturnFromEditor()
         {
             tutorialGUI.EventReceiver?.OnTutorialDisabled();
         }
+        else
+        {
+            // Show day/night cycle tutorial when entering a patch with sunlight
+            if (GameWorld.WorldSettings.DayNightCycleEnabled)
+            {
+                var sunlight = SimulationParameters.Instance.GetCompound("sunlight");
+                var patchSunlight = GameWorld.Map.CurrentPatch!.GetCompoundAmount(sunlight, CompoundAmountType.Biome);
+
+                if (patchSunlight > 0.01f)
```

Comment:
One more thing, can you make `0.01f` a constant field in `Constants`?

Reply:
This would be nice to name. I think right now this is picked so that there is some effect during the night?

Reply:
I picked it so that the tutorial wouldn't show up in 1% light levels, as it is unlikely people will develop photosynthesis in those areas